pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -319,12 +318,31 @@ var legacyRemoteSchema = map[string]*schema.Schema{
 		Type:     schema.TypeList,
 		Optional: true,
 		Computed: true,
-		MaxItems: 1,
```

Comment:
We don't need to constraint to one item any more? How would that work if user defines more than one in HCL?

Reply:
Will limit to 1. 
Assumed that limit was for attributes under 'content_synchronisation' but it was for 'content_synchronisation' block itself.